### ISSUE TITLE
Update Step-by-step template to support unlimited field cardinality

### ISSUE THREAD

**Issue by oksana-c** [department-of-veterans-affairs/va.gov-team#14197]
## Issue Description

`field_steps` cardinality will be set to Unlimited in [3041](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/3041)

## Acceptance Criteria

- [ ] Step-by-step metalsmith template is updated to support unlimited cardinality for `field_steps`

**Comment by oksana-c** [comment]
@ncksllvn @kelsonic per our chat today, we'll be switching field_steps cardinality to unlimited. This issue is to cover follow-up work needed for step-by-step template.

cc @MickinSahni @brianalloyd

**Comment by kelsonic** [comment]
PR merged, ready for validation at http://preview-prod.vfs.va.gov/preview?nodeId=6899 following today's deploy (or Monday's as I think I just missed the cutoff)

**Comment by jenniferlee-dsva** [comment]
Hi - I see the steps now @kelsonic!  To make sure the new cardinality function is working, I added a part 2 step-by-step section, and that doesn't appear in the preview however. So I don't think the cardinality function (allowing more than 1 subway map block on page) is working. 

Separately, there are several test in-body alerts that @RLHecht and I have added within some of the steps, and those also are not displaying. (That's part of the standard template requirements, separate from 'cardinality' that allows more than one subway map block on the same page.)

For reference: 
https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/content/tier-2-content-IA-and-design/learning-center-mvp/template-requirements.md#step-by-step 
![image](https://user-images.githubusercontent.com/43381063/95870967-1dda2800-0d3b-11eb-8288-8f99de3a3f2d.png)
